propertysuggester/parser/CsvWriter.py

```diff
@@ -20,15 +20,16 @@ def write_csv(entities, output_file, delimiter=","):
                 write_row(csv_writer, title, "reference", ref)
 
 
-def write_row(csv_writer, title, typ, snak):
+def write_row(csv_writer, title, type_, snak):
```

Comment:
https://www.python.org/dev/peps/pep-0008/#function-and-method-arguments

Reply:
"If a function argument's name clashes with a reserved keyword, it is generally better to append a single trailing underscore rather than use an abbreviation or spelling corruption. Thus class_ is better than clss. (Perhaps better is to avoid such clashes by using a synonym.)"

Reply:
you de-germanized it!